flexmeasures/ui/templates/views/sensors.html

```diff
@@ -25,40 +25,167 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
           <div class="container">
-            <div class="row">
-                <h3>{{ sensor.name }}</h3>
-                <div class="col-sm-6">
-                    <h5>Properties</h5>
-
-                    <table class="table table-striped">
-                        <tr>
-                            <th>Unit</th>
-                            <td>{{ sensor.unit }}</td>
-                        </tr>
-                        <tr>
-                            <th>Event Resolution</th>
-                            <td>{{ sensor.event_resolution }}</td>
-                        </tr>
-                        <tr>
-                            <th>Timezone</th>
-                            <td>{{ sensor.timezone }}</td>
-                        </tr>
-                        <tr>
-                            <th>Knowledge Horizon Type</th>
-                            <td>{{ sensor.knowledge_horizon_fnc }}</td>
-                        </tr>
-                    </table>
+                <div class="row">
+                    <h3>{{ sensor.name }}</h3>
+                    <div class="col-sm-6" id="propertiesContainer">
+                        <h5>Properties</h5>
+                        <table class="table table-striped">
+                            <tr>
+                                <th>Unit</th>
+                                <td>{{ sensor.unit }}</td>
+                            </tr>
+                            <tr>
+                                <th>Event Resolution</th>
+                                <td>{{ sensor.event_resolution }}</td>
+                            </tr>
+                            <tr>
+                                <th>Timezone</th>
+                                <td>{{ sensor.timezone }}</td>
+                            </tr>
+                            <tr>
+                                <th>Knowledge Horizon Type</th>
+                                <td>{{ sensor.knowledge_horizon_fnc }}</td>
+                            </tr>
+                        </table>
+                    </div>
+                    <div class="col-sm-6" id="statsContainer">
+                        <span id="spinner-run-simulation" class="spinner-border spinner-border-sm d-none" role="status"></span>
+                        <h5 id="statsHeader">Statistics</h5>
+                        <table id="statsTable" class="table table-striped">
+                            <tbody id="statsTableBody">
+                            </tbody>
+                        </table>
+                        <!-- Dropdown for sourceKey -->
+                        <div class="dropdown mb-3 d-none" id="sourceKeyDropdownContainer">
+                            <h6>Select source for statistics</h6>
```

Comment:
Please save vertical space and put this just before the dropdown. Maybe with `<small>`?